v1/pipeline/pipeline_using_template_spec.yaml

```diff
@@ -0,0 +1,27 @@
+type: object
+title: PipelineUsingTemplateSpec
+required:
+  - id
+  - versionLabel
+properties:
+  id:
+    type: string
+    description: id of template to be linked in pipeline
+    pattern: "^[a-zA-Z_][0-9a-zA-Z_]{0,127}$"
+  versionLabel:
+    type: string
+    pattern: "^[0-9a-zA-Z][^\\s/&]{0,127}$"
+    description: version label of template to be linked in pipeline
+  template_inputs:
```

Comment:
Can we just call them inputs?

Reply:
yeah, I think we can use just inputs here